name_color_codes.csv

```diff
@@ -1,18 +1,18 @@
 code,name_english,name_spanish,colour
 BIO,Biomass/Biofuels,Biomasa,forestgreen
-COA,Coal,Carb�n,black
-CRU,Crude oil,Petr�leo crudo,red
-DSL,Diesel,Di�sel,orange
```

Comment:
This looks off because we're forcing the encoding to `latin-1` here: https://github.com/ClimateCompatibleGrowth/osemosys-cloud-visualization/blob/34ef8fa3a97fb6e69ffd2efc6f52f4a28678e16b/app/constants.py#L17

I'm worried that if we change this in the file, the file encodings will start causing issues (meaning we see the correct characters in the code but in the app they will be wrong).

The safest way is to keep this file in `latin-1` and change the encoding in your text editor (see for example https://developerf1.com/snippet/change-file-encoding-in-vs-code for VS code).